tests/test_definitions/column_map_expectations/expect_column_values_to_be_unique.json

```diff
@@ -7,7 +7,7 @@
       "c" : [1, 1, 2, null],
       "d" : [1, "1", 2, null],
       "n" : [null, null, 2, null],
-      "unique" : [1, 2, 3, null],
```

Comment:
This is due to a really strange error that I'm getting only on the Azure pipelines. `basic_positive_test_case` is the only test that refers to this column ... the test will still test a "basic" positive test case...